protocol/app/upgrades/v0.2.2/constants.go

```diff
@@ -0,0 +1,23 @@
+package v0_2_2
+
+import (
+	store "github.com/cosmos/cosmos-sdk/store/types"
+	"github.com/dydxprotocol/v4-chain/protocol/app/upgrades"
+)
+
+const UpgradeName = "v0.2.2"
+
+var (
+	Fork = upgrades.Fork{
+		UpgradeName: UpgradeName,
+		// Target upgrade time is Sept 15, 2023, 1pm EST,
+		// estimated to occur on block 178500, assuming 1.5s block time.
```

Comment:
Meganit: current testnet block time around [1.4s](https://app.datadoghq.com/dashboard/wr9-2px-quy/v4?refresh_mode=sliding&tpl_var_env%5B0%5D=testnet&from_ts=1694636335033&to_ts=1694647135033&live=false&tile_focus=8932930888001242), up to you whether to upate height accordingly